gems/aws-sdk-s3/lib/aws-sdk-s3/file_uploader.rb

```diff
@@ -37,7 +37,7 @@ def initialize(options = {})
       #   objects smaller than the multipart threshold.
       # @return [void]
       def upload(source, options = {})
-        Aws::Plugins::UserAgent.feature('s3-transfer') do
+        Aws::Plugins::UserAgent.metric('S3_TRANSFER') do
```

Comment:
I recall hearing that there was some differences with our implementation of "S3 Transfer Manager" vs the most recently defined specifications. Even so, we are still collecting the metrics under this ID, right?

Reply:
Yeah. This is effectively the same feature and we should track it.